### PR TITLE
Update react-webpack.md

### PR DESCRIPTION
**Pull request by daslicht**
Fixes #

see : https://github.com/typings/typings/issues/559#issuecomment-222202351

### CODE REVIEW

**Comment by daslicht** [comment]
see: https://github.com/typings/typings/issues/559#issuecomment-222202351

**Comment by daslicht** [comment]
there seam to be more infound outdated , I get alot other errors . frustrating

**Comment by daslicht** [comment]
```
daslicht:react-ts-webpack daslicht$ webpack
ts-loader: Using typescript@1.8.10
Hash: 1a6de37ace656ee34aba
Version: webpack 1.13.1
Time: 865ms
    + 1 hidden modules

ERROR in /Users/daslicht/DEV/Experiments/react-ts-webpack/src/components/Hello.tsx
(1,24): error TS2307: Cannot find module 'react'.

ERROR in /Users/daslicht/DEV/Experiments/react-ts-webpack/src/components/Hello.tsx
(7,16): error TS17004: Cannot use JSX unless the '--jsx' flag is provided.

ERROR in /Users/daslicht/DEV/Experiments/react-ts-webpack/src/components/Hello.tsx
(7,37): error TS2339: Property 'props' does not exist on type 'Hello'.

ERROR in /Users/daslicht/DEV/Experiments/react-ts-webpack/src/components/Hello.tsx
(7,63): error TS2339: Property 'props' does not exist on type 'Hello'.

ERROR in ./src/index.tsx
(1,24): error TS2307: Cannot find module 'react'.

ERROR in ./src/index.tsx
(2,27): error TS2307: Cannot find module 'react-dom'.

ERROR in ./src/index.tsx
(7,5): error TS17004: Cannot use JSX unless the '--jsx' flag is provided.

ERROR in ./src/index.tsx
Module parse failed: /Users/daslicht/DEV/Experiments/react-ts-webpack/node_modules/ts-loader/index.js!/Users/daslicht/DEV/Experiments/react-ts-webpack/src/index.tsx Unexpected token (5:16)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (5:16)
    at Parser.pp.raise (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:920:13)
    at Parser.pp.unexpected (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1483:8)
    at Parser.pp.parseExprAtom (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:330:12)
    at Parser.pp.parseExprSubscripts (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:225:19)
    at Parser.pp.parseMaybeUnary (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:204:17)
    at Parser.pp.parseExprOps (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:151:19)
    at Parser.pp.parseMaybeConditional (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:133:19)
    at Parser.pp.parseMaybeAssign (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:110:19)
    at Parser.pp.parseExprList (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:657:23)
    at Parser.pp.parseSubscripts (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:249:29)
    at Parser.pp.parseExprSubscripts (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:228:15)
    at Parser.pp.parseMaybeUnary (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:204:17)
    at Parser.pp.parseExprOps (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:151:19)
    at Parser.pp.parseMaybeConditional (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:133:19)
    at Parser.pp.parseMaybeAssign (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:110:19)
    at Parser.pp.parseExpression (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:86:19)
    at Parser.pp.parseStatement (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1753:23)
    at Parser.pp.parseTopLevel (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1648:21)
    at Parser.parse (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:1616:17)
    at Object.parse (/usr/local/lib/node_modules/webpack/node_modules/acorn/dist/acorn.js:882:44)
    at Parser.parse (/usr/local/lib/node_modules/webpack/lib/Parser.js:902:15)
    at DependenciesBlock.<anonymous> (/usr/local/lib/node_modules/webpack/lib/NormalModule.js:104:16)
    at DependenciesBlock.onModuleBuild (/usr/local/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:310:10)
    at nextLoader (/usr/local/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:275:25)
    at /usr/local/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:292:15
    at context.callback (/usr/local/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:148:14)
    at Object.loader (/Users/daslicht/DEV/Experiments/react-ts-webpack/node_modules/ts-loader/index.js:468:5)
    at WEBPACK_CORE_LOADER_EXECUTION (/usr/local/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:155:71)
    at runSyncOrAsync (/usr/local/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:155:93)
    at nextLoader (/usr/local/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:290:3)
daslicht:react-ts-webpack daslicht$ 

```

**Comment by daslicht** [comment]
thats what I have : https://github.com/daslicht/react-webpack-ts

File: pages/quick-start/react-webpack.md [hunk 1]
```diff
@@ -83,8 +83,8 @@ If you want a local copy, just run `npm install typescript`.
 Finally, we'll use Typings to grab the declaration files for React and ReactDOM:
 
 ```shell
-typings install --ambient --save react
-typings install --ambient --save react-dom
+typings install dt~react --global --save
+typings install dt~react-dom --global --save
 ```
 
 The `--ambient` flag will tell Typings to grab any declaration files from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped), a repository of community-authored `.d.ts` files.
```
Comment (blakeembrey) [inline pages/quick-start/react-webpack.md]: The `--global` flag tells Typings to install the declaration files as-is, as the declaration files alter the global compiler namespace.

**Comment by blakeembrey** [comment]
Files needs to change from `typings/main.d.ts` to `typings/index.d.ts` also. Only "main" is output by default now.

File: pages/quick-start/react-webpack.md (hunk 1 above)
Comment (daslicht) [inline pages/quick-start/react-webpack.md]: this did not work anyways:
```
typings install --ambient --save react
 typings install --ambient --save react-dom
```

**Comment by blakeembrey** [comment]
@daslicht You need to create the `tsconfig.json` file.

**Comment by daslicht** [comment]
see : https://github.com/daslicht/react-webpack-ts/blob/master/typings.json#L11
already have that

**Comment by blakeembrey** [comment]
@daslicht See my message above. The comment there was related to the instructions you've altered, not your particular setup - there's a few more things that need changing to support Typings 1.0.

File: pages/quick-start/react-webpack.md (hunk 1 above)
Comment (blakeembrey) [inline pages/quick-start/react-webpack.md]: Again. This is a comment for what else needs to change in the instructions.

**Comment by daslicht** [comment]
Fixed :) I mixed up tsconfig with the typings !

File: pages/quick-start/react-webpack.md (hunk 1 above)
Comment (daslicht) [inline pages/quick-start/react-webpack.md]: aye aye

**Comment by blakeembrey** [comment]
In the instructions also, it'd be good to not recommend using `npm link`. That's a bad practice here, people should be installing a local copy of `typescript`. 

@daslicht Not related to your PR/issues in general again, just some observations from reading these instructions 😄

**Comment by blakeembrey** [comment]
@daslicht Awesome, great to hear 😄

**Comment by daslicht** [comment]
@blakeembrey hope to see you some day in irc ;)

**Comment by blakeembrey** [comment]
Which IRC channel is that? I haven't tried IRC in so long.

**Comment by daslicht** [comment]
freenode #typescript or do you prefer gitter or slack ?

**Comment by DanielRosenwasser** [comment]
Sorry, [the guide is already fixed](https://github.com/Microsoft/TypeScript-Handbook/blob/f36b33012d7d03829cb7185964ceabe3a29daf8f/pages/tutorials/React%20%26%20Webpack.md) but the website hasn't been updated yet. I'll try to get an update out soon, but I'm going to close this PR.

**Comment by blakeembrey** [comment]
> freenode #typescript or do you prefer gitter or slack ?

I guess I'm logged in on Gitter and Slack, but some channels I just can't keep up with - one of those is the TypeScript channel too...

**Pull request closed without merging**